handler/src/main/java/io/netty/handler/ssl/SslContextBuilder.java

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.handler.ssl;
+
+import static io.netty.util.internal.ObjectUtil.checkNotNull;
+
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLException;
+import javax.net.ssl.TrustManagerFactory;
+import java.io.File;
+
+/**
+ * Builder for configuring a new SslContext for creation.
+ */
+public final class SslContextBuilder {
+    /**
+     * Creates a builder for new client-side {@link SslContext}.
+     */
+    public static SslContextBuilder forClient() {
```

Comment:
I think it would read weird to have a non-getter/setter be named a noun. If I just read ```SslContextBuilder.client().build()``` I would assume client() was some getter that got a static instance or some such.